packages/components/calendar/src/range-date-picker.tsx

```diff
@@ -234,7 +234,7 @@ export const RangeDatePickerField = forwardRef<
         >
           <AutosizingInput
             ref={mergeRefs(ref, startInputRef)}
-            className="ui-range-date-picker__field__start-input"
+            className="ui-range-date-picker__field-input--start"
```

Comment:
I'm wondering if it's OK to use modifiers without the corresponding original class, `ui-range-date-picker__field-input` in this case.

Without the original class, it's like modifiers _modify_ nothing?

```suggestion
            className="ui-range-date-picker__field-input ui-range-date-picker__field-input--start"
```

Reply:
I think than you are right.
I will fix it.
Thanks.

Reply:
Donne